src/curated-corpus/components/AddProspectForm/AddProspectForm.test.tsx

```diff
@@ -95,71 +63,4 @@ describe('The AddProspectForm component', () => {
 
     expect(incorrectUrlError).toBeInTheDocument();
   });
-
-  it('should render approved item modal when a correct url is provided', async () => {
-    // mock meta data object that will be used in the apollo provider mock
-    // to be returned as one of the query results
-    const mockUrlMetadata = {
```

Comment:
I am sad to see this test go but it's been made redundant by the refactor. I have made a note of this and am hoping to incorporate it back when we have time to write integration tests for page-level components.